java/parser/src/main/java/org/sqlflow/parser/parse/ParseInterface.java

```diff
@@ -0,0 +1,15 @@
+package org.sqlflow.parser.parse;
+
+public interface ParseInterface {
+  // ParseAndSplit calls the underlining parser to parse a SQL program and returns a ParseResult.
+  //
+  // It returns <statements, -1, ""> if Calcite parser accepts the SQL program.
+  //     input:  "select 1; select 1;"
+  //     output: {"select 1;", "select 1;"}, -1 , nil
+  // It returns <statements, idx, ""> if Calcite parser accepts part of the SQL program,
+  // indicated by idx.
+  //     input:  "select 1; select 1 to train; select 1"
+  //     output: {"select 1;", "select 1"}, 19, nil
+  // It returns <nil, -1, error> if an error is occurred.
+  ParseResult ParseAndSplit(String sql);
```

Comment:
Change `ParseAndSplit` to`parseAndSplit` or `parse`

Reply:
You are right. Function naming should be camelCase. I will fix it in the following up PR.